go/serialization_test.go

```diff
@@ -203,112 +203,151 @@ const (
 		"0000" +
 		// extensions, 0 bytes
 		""
+
+	defaultSTHSignedHexString string =
+	// version, 1 byte
+	"00" +
+		// signature type, 1 byte
+		"01" +
+		// timestamp, 8 bytes
+		"0000000000000929" +
+		// tree size, 8 bytes
+		"0000000000000006" +
+		// root hash, 32 bytes
+		"696d757374626565786163746c7974686972747974776f62797465736c6f6e67"
 )
 
-func DefaultSCTLogID() []byte {
-	return []byte(DefaultSCTLogIDString)
+func defaultSCTLogID() []byte {
+	return []byte(defaultSCTLogIDString)
```

Comment:
Is it not possible to do the cast while defining the const? (Asked from a position of complete ignorance)